packages/form-js-editor/src/features/properties-panel/entries/factories/zeroPositiveIntegerEntryFactory.js

```diff
@@ -0,0 +1,61 @@
+import { get } from 'min-dash';
+import { useService } from '../../hooks';
+import { NumberFieldEntry, isNumberFieldEntryEdited } from '@bpmn-io/properties-panel'; // Assuming NumberFieldEntry exists similar to TextFieldEntry
```

Comment:
Can we remove this comment? Or did you inspect any missing features?

Reply:
@Skaiir ☝️